themes/bootstrap3/templates/record/comments-list.phtml

```diff
@@ -5,20 +5,20 @@
   <?php foreach ($comments as $comment): ?>
     <div class="comment">
       <div class="comment-name">
-        <strong><?=null === $comment->user_id ? $this->transEsc('comment_anonymous_user') : $this->escapeHtml(trim($comment->firstname . ' ' . $comment->lastname))?></strong><br/>
+        <strong><?=null === $comment[0]->getUser()->getId() ? $this->transEsc('comment_anonymous_user') : $this->escapeHtml(trim($comment['firstname'] . ' ' . $comment['lastname']))?></strong><br/>
```

Comment:
I understand why the `$comment[0]` syntax is necessary here, but it's a bit awkward to read and write. What array keys are available here? Is there a named equivalent to `$comment[0]` like `$comment['comment']`? Are there numeric equivalents to `firstname` and `lastname`? Depending on how things are structured, I'm wondering if it might be better to structure the code more like:

```
<?php foreach ($comments as $commentRow): ?>
  <?php [$comment, $firstname, $lastname] = $commentRow; ?>
...
```

So that we destructure the row at the top of the loop, and then can use more readable variables in the subsequent code. But maybe the data structure doesn't make this easy.